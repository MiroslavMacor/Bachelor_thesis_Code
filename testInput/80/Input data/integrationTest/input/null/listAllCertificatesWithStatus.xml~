<?xml version="1.0"?><remsig><status></status></remsig>
