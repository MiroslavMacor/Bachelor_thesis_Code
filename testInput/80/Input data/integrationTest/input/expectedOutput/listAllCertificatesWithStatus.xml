<?xml version="1.0" encoding="UTF-8"?><remsig>  <certificate id="1" userId="1235">    <dn>1.2.840.113549.1.9.2=#1606343039373831,CN=Silvia Vigasova,O=Masarykova univerzita,C=CZ</dn>    <issuer>CN=TERENA Personal CA,O=TERENA,C=NL</issuer>    <serialNumber>185871382282857173108224673686393163453</serialNumber>    <expirationFrom>1408924800</expirationFrom>    <expirationTo>1503619199</expirationTo>    <qualified>0</qualified>    <certificatePEM>-----BEGIN CERTIFICATE-----MIIEljCCA36gAwIBAgIRAIvVhq4eC8WdKCy1wh0njr0wDQYJKoZIhvcNAQEFBQAwOzELMAkGA1UEBhMCTkwxDzANBgNVBAoTBlRFUkVOQTEbMBkGA1UEAxMSVEVSRU5BIFBlcnNvbmFsIENBMB4XDTE0MDgyNTAwMDAwMFoXDTE3MDgyNDIzNTk1OVowXjELMAkGA1UEBhMCQ1oxHjAcBgNVBAoTFU1hc2FyeWtvdmEgdW5pdmVyeml0YTEYMBYGA1UEAxMPU2lsdmlhIFZpZ2Fzb3ZhMRUwEwYJKoZIhvcNAQkCFgY0MDk3ODEwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDDxxVXsAPUl7uzUvGNZZK720LDG09aFmkjo/DFDHCa9wgV1IrjReCTMQ9N3rqqqsMcr4auBV5MfcAE+dgKccIvnazl0ldWTSlViWidMTWPKSqjmxByKzDmB7yebaia15X+2pdf7YCy8fXX8aCqJyl24iKeK8krdZI8HBXMFgfVkf1ass+Ia+MDqWR0qtsTwq7P3I/XPNA1dtM55hrtNYP2cP287AJxNjP02YAc27vSUfv33p3PkIILQPu4GFvfG4FoM2nGGdmj4el5qlSasne78cahijh5yuxQ7EE7c9Am8XkoTcy6TpKQWsuO3KdL2xpSfARrSuVw/JUmzTVZrmyxAgMBAAGjggFwMIIBbDAfBgNVHSMEGDAWgBRjTUNaGUg/xEbBArq/7g7lgrdmpjAdBgNVHQ4EFgQUTIASGncjlZNI0uEjVzztIM7Esq4wDgYDVR0PAQH/BAQDAgWgMAwGA1UdEwEB/wQCMAAwHQYDVR0lBBYwFAYIKwYBBQUHAwQGCCsGAQUFBwMCMBgGA1UdIAQRMA8wDQYLKwYBBAGyMQECAh0wPwYDVR0fBDgwNjA0oDKgMIYuaHR0cDovL2NybC50Y3MudGVyZW5hLm9yZy9URVJFTkFQZXJzb25hbENBLmNybDByBggrBgEFBQcBAQRmMGQwOgYIKwYBBQUHMAKGLmh0dHA6Ly9jcnQudGNzLnRlcmVuYS5vcmcvVEVSRU5BUGVyc29uYWxDQS5jcnQwJgYIKwYBBQUHMAGGGmh0dHA6Ly9vY3NwLnRjcy50ZXJlbmEub3JnMB4GA1UdEQQXMBWBEzQwOTc4MUBtYWlsLm11bmkuY3owDQYJKoZIhvcNAQEFBQADggEBADLxnpkgvNhMWurNfqv4RA3eoyeq8et1AOJZ4/4jdCVBCGu88FEmXRt3WJEwbo+m5Sc8Fg1masU4mi51k5HnavlRJU7AXI2pgDjQB+tLx2BfMYTmdIf0UkSVfaagQOWpJ51w/V3sDDiuBEV5wD9ZlisUA3iT+cD75cPC6oD3bZECNCicm5r0xaISwfu+C9flI08Gs6T03RQwZ1VUjqbqaN9jnwPTK+Oe5laf70dGcqteHaEKjbnrdTvScPfg/kIJllWujaNfgltill9T15bCK8fauamlLvJDx1cz5TycSXXoJnCSJhCqUGWL60PIs3VCow/yh/BjEJgwGSwLqmfRpiA=-----END CERTIFICATE-----</certificatePEM>    <chainPEM>-----BEGIN CERTIFICATE-----MIIEljCCA36gAwIBAgIRAIvVhq4eC8WdKCy1wh0njr0wDQYJKoZIhvcNAQEFBQAwOzELMAkGA1UEBhMCTkwxDzANBgNVBAoTBlRFUkVOQTEbMBkGA1UEAxMSVEVSRU5BIFBlcnNvbmFsIENBMB4XDTE0MDgyNTAwMDAwMFoXDTE3MDgyNDIzNTk1OVowXjELMAkGA1UEBhMCQ1oxHjAcBgNVBAoTFU1hc2FyeWtvdmEgdW5pdmVyeml0YTEYMBYGA1UEAxMPU2lsdmlhIFZpZ2Fzb3ZhMRUwEwYJKoZIhvcNAQkCFgY0MDk3ODEwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDDxxVXsAPUl7uzUvGNZZK720LDG09aFmkjo/DFDHCa9wgV1IrjReCTMQ9N3rqqqsMcr4auBV5MfcAE+dgKccIvnazl0ldWTSlViWidMTWPKSqjmxByKzDmB7yebaia15X+2pdf7YCy8fXX8aCqJyl24iKeK8krdZI8HBXMFgfVkf1ass+Ia+MDqWR0qtsTwq7P3I/XPNA1dtM55hrtNYP2cP287AJxNjP02YAc27vSUfv33p3PkIILQPu4GFvfG4FoM2nGGdmj4el5qlSasne78cahijh5yuxQ7EE7c9Am8XkoTcy6TpKQWsuO3KdL2xpSfARrSuVw/JUmzTVZrmyxAgMBAAGjggFwMIIBbDAfBgNVHSMEGDAWgBRjTUNaGUg/xEbBArq/7g7lgrdmpjAdBgNVHQ4EFgQUTIASGncjlZNI0uEjVzztIM7Esq4wDgYDVR0PAQH/BAQDAgWgMAwGA1UdEwEB/wQCMAAwHQYDVR0lBBYwFAYIKwYBBQUHAwQGCCsGAQUFBwMCMBgGA1UdIAQRMA8wDQYLKwYBBAGyMQECAh0wPwYDVR0fBDgwNjA0oDKgMIYuaHR0cDovL2NybC50Y3MudGVyZW5hLm9yZy9URVJFTkFQZXJzb25hbENBLmNybDByBggrBgEFBQcBAQRmMGQwOgYIKwYBBQUHMAKGLmh0dHA6Ly9jcnQudGNzLnRlcmVuYS5vcmcvVEVSRU5BUGVyc29uYWxDQS5jcnQwJgYIKwYBBQUHMAGGGmh0dHA6Ly9vY3NwLnRjcy50ZXJlbmEub3JnMB4GA1UdEQQXMBWBEzQwOTc4MUBtYWlsLm11bmkuY3owDQYJKoZIhvcNAQEFBQADggEBADLxnpkgvNhMWurNfqv4RA3eoyeq8et1AOJZ4/4jdCVBCGu88FEmXRt3WJEwbo+m5Sc8Fg1masU4mi51k5HnavlRJU7AXI2pgDjQB+tLx2BfMYTmdIf0UkSVfaagQOWpJ51w/V3sDDiuBEV5wD9ZlisUA3iT+cD75cPC6oD3bZECNCicm5r0xaISwfu+C9flI08Gs6T03RQwZ1VUjqbqaN9jnwPTK+Oe5laf70dGcqteHaEKjbnrdTvScPfg/kIJllWujaNfgltill9T15bCK8fauamlLvJDx1cz5TycSXXoJnCSJhCqUGWL60PIs3VCow/yh/BjEJgwGSwLqmfRpiA=-----END CERTIFICATE-----</chainPEM>  </certificate>  <certificate id="7" userId="1236">    <dn>CN=sub1,OU=testDep,O=remsigTest,ST=czechRepublik,C=cz</dn>    <issuer>CN=greatTester,C=cz,ST=czechRepublik,L=brno,1.2.840.113549.1.9.1=#160f74657374657231406963732e636f6d,OU=testDep,O=remsigTest</issuer>    <serialNumber>1048577</serialNumber>    <expirationFrom>1455623422</expirationFrom>    <expirationTo>1518695422</expirationTo>    <qualified>0</qualified>    <certificatePEM>-----BEGIN CERTIFICATE-----MIIFYzCCA0sCAxAAATANBgkqhkiG9w0BAQQFADCBkTETMBEGA1UEChMKcmVtc2lnVGVzdDEQMA4GA1UECxMHdGVzdERlcDEeMBwGCSqGSIb3DQEJARYPdGVzdGVyMUBpY3MuY29tMQ0wCwYDVQQHEwRicm5vMRYwFAYDVQQIEw1jemVjaFJlcHVibGlrMQswCQYDVQQGEwJjejEUMBIGA1UEAxMLZ3JlYXRUZXN0ZXIwHhcNMTYwMjE2MTE1MDIyWhcNMTgwMjE1MTE1MDIyWjBbMQswCQYDVQQGEwJjejEWMBQGA1UECBMNY3plY2hSZXB1YmxpazETMBEGA1UEChMKcmVtc2lnVGVzdDEQMA4GA1UECxMHdGVzdERlcDENMAsGA1UEAxMEc3ViMTCCAiIwDQYJKoZIhvcNAQEBBQADggIPADCCAgoCggIBANRmKZ69IM89ZppfE7wyW7vQun58pb4PEOQPxSLJ+vb4duxBbKGCSq/G4qaapUJLu84Z5TNsziROniTYSm24+1hE/DMhf73uYeU1UF1tSecEb/anu2u6N6t5ojIImKI5XZnGIeJOCYaFdRMnLaSYEGPf4tyvLwPsc1dyC1Sn4r//Iqs+sXyWjK17yZfJGUjoo27WuS+i5DBmqXUpJr7222P9d+idgmQBpxwrrS77Q9rpUNDnUB4oT0hewKgWE6wI/HQ0C7t4MAhMi34XKKm+V+vmwcvwYm8x5RpcR81XaoaWMh3Dtl52LOiJh9JKzs3GgNkGVSdX3OonxhYk4YGFf/5uP/2wP8Z57wP0fjsat8jV3B94B8ePBT5THJEuNOqBhtCGR3Z+45LvV2wvLaMf/DsjLachVW/BUokigwCfUPpofeVURPn5+4nNf0tubJBQWE3MvHUUqDo7kKLWQbUlMr3ECmQqX10wSUrgwZ1LNk2auOCaOupwyZSg+bVzhC9d9OGcEImE4VPk+KSZkPJSoK65tXwVFV4Cor83SupeIeh5UHGS1VnJWkvEv6i3lsdkiuyVPmfXWCXvTfQoobRJ9fH0PJp5HcPx1+lfxziOTRNX1yLKQjxpf0V9O6245GgIMZDQQUNbehAEiZQ+DYF24pkjpSbYAFv+Ah7foVER2/5JAgMBAAEwDQYJKoZIhvcNAQEEBQADggIBAAuz/dz0LjaRuMVZojBw6cd7JKGjFsbJ8TbOewl48T0GDT8kDwd3RHIyJvU2BetHWAYoHCpx/+Wb+aIbg5N4sfcSaLA2wntUIjsMAFDY3mhJtPBlwpuJC4NECtrKn481OiIfzmjgrf2ahq8nH6dItgRZgK1DwXuOAdhkAHso8/sAWCYm+aIrr+HZw2aDk5TspBw/JWnEHt2fcryv9wxTMvKiHxCDe4wsGnQDqp4XHURqQMgI2caiIE/NTfgH6Al1XjktG+hAJdB7bGH9nsX5JTtAVkW5o0GKlqgrlSsKGsEvv8zfzGxmM+F02ZAFG/8XCvaUJJ4kX4k8iM0m22tTrcg/hjlMJOSs7ceHPD4x30R/+Ue20czoE+YGHCQiOVkY9oIFG/yq0DZW5QAYcJM8KsEuwa09SnK0jPR29/59RWpJsW8GxtXWaibjtjjcPTIaw2Fsfxuckzy1SupqEC45UetE0hs/ytnE1N6sIGiX14PqjmOrexOko04RAdCUYO1LgHrBfGv83zl8hxgnKo2Y8OA6YxGAB+W1Ezw6zfNQWNFaoNsZEvuphH/ChCg8q+n365/g/Hc1pP6dd9nAPKGV6+HTLVMu9frZKP3mUZ1Wsq2Q0n4nsQ262R+949BZWCHKeYeaw015t//WkDWxYcV5WUN3WBSLtsE/mtOUBCmw6Zj/-----END CERTIFICATE-----</certificatePEM>    <chainPEM>-----BEGIN CERTIFICATE-----MIIFYzCCA0sCAxAAATANBgkqhkiG9w0BAQQFADCBkTETMBEGA1UEChMKcmVtc2lnVGVzdDEQMA4GA1UECxMHdGVzdERlcDEeMBwGCSqGSIb3DQEJARYPdGVzdGVyMUBpY3MuY29tMQ0wCwYDVQQHEwRicm5vMRYwFAYDVQQIEw1jemVjaFJlcHVibGlrMQswCQYDVQQGEwJjejEUMBIGA1UEAxMLZ3JlYXRUZXN0ZXIwHhcNMTYwMjE2MTE1MDIyWhcNMTgwMjE1MTE1MDIyWjBbMQswCQYDVQQGEwJjejEWMBQGA1UECBMNY3plY2hSZXB1YmxpazETMBEGA1UEChMKcmVtc2lnVGVzdDEQMA4GA1UECxMHdGVzdERlcDENMAsGA1UEAxMEc3ViMTCCAiIwDQYJKoZIhvcNAQEBBQADggIPADCCAgoCggIBANRmKZ69IM89ZppfE7wyW7vQun58pb4PEOQPxSLJ+vb4duxBbKGCSq/G4qaapUJLu84Z5TNsziROniTYSm24+1hE/DMhf73uYeU1UF1tSecEb/anu2u6N6t5ojIImKI5XZnGIeJOCYaFdRMnLaSYEGPf4tyvLwPsc1dyC1Sn4r//Iqs+sXyWjK17yZfJGUjoo27WuS+i5DBmqXUpJr7222P9d+idgmQBpxwrrS77Q9rpUNDnUB4oT0hewKgWE6wI/HQ0C7t4MAhMi34XKKm+V+vmwcvwYm8x5RpcR81XaoaWMh3Dtl52LOiJh9JKzs3GgNkGVSdX3OonxhYk4YGFf/5uP/2wP8Z57wP0fjsat8jV3B94B8ePBT5THJEuNOqBhtCGR3Z+45LvV2wvLaMf/DsjLachVW/BUokigwCfUPpofeVURPn5+4nNf0tubJBQWE3MvHUUqDo7kKLWQbUlMr3ECmQqX10wSUrgwZ1LNk2auOCaOupwyZSg+bVzhC9d9OGcEImE4VPk+KSZkPJSoK65tXwVFV4Cor83SupeIeh5UHGS1VnJWkvEv6i3lsdkiuyVPmfXWCXvTfQoobRJ9fH0PJp5HcPx1+lfxziOTRNX1yLKQjxpf0V9O6245GgIMZDQQUNbehAEiZQ+DYF24pkjpSbYAFv+Ah7foVER2/5JAgMBAAEwDQYJKoZIhvcNAQEEBQADggIBAAuz/dz0LjaRuMVZojBw6cd7JKGjFsbJ8TbOewl48T0GDT8kDwd3RHIyJvU2BetHWAYoHCpx/+Wb+aIbg5N4sfcSaLA2wntUIjsMAFDY3mhJtPBlwpuJC4NECtrKn481OiIfzmjgrf2ahq8nH6dItgRZgK1DwXuOAdhkAHso8/sAWCYm+aIrr+HZw2aDk5TspBw/JWnEHt2fcryv9wxTMvKiHxCDe4wsGnQDqp4XHURqQMgI2caiIE/NTfgH6Al1XjktG+hAJdB7bGH9nsX5JTtAVkW5o0GKlqgrlSsKGsEvv8zfzGxmM+F02ZAFG/8XCvaUJJ4kX4k8iM0m22tTrcg/hjlMJOSs7ceHPD4x30R/+Ue20czoE+YGHCQiOVkY9oIFG/yq0DZW5QAYcJM8KsEuwa09SnK0jPR29/59RWpJsW8GxtXWaibjtjjcPTIaw2Fsfxuckzy1SupqEC45UetE0hs/ytnE1N6sIGiX14PqjmOrexOko04RAdCUYO1LgHrBfGv83zl8hxgnKo2Y8OA6YxGAB+W1Ezw6zfNQWNFaoNsZEvuphH/ChCg8q+n365/g/Hc1pP6dd9nAPKGV6+HTLVMu9frZKP3mUZ1Wsq2Q0n4nsQ262R+949BZWCHKeYeaw015t//WkDWxYcV5WUN3WBSLtsE/mtOUBCmw6Zj/-----END CERTIFICATE-----</chainPEM>  </certificate>  <operationId>1463326314552</operationId></remsig>
